app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -31,17 +46,250 @@ class MainActivity : ComponentActivity() {
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
+fun CardAssignment(modifier: Modifier = Modifier) {
+    Box(
         modifier = modifier
-    )
+            .padding(all = 45.dp)
+            .size(width = 240.dp, height = 140.dp)
+            .border(
+                width = 1.dp,
+                color = Color.Black,
+                shape = RoundedCornerShape(size = 10.dp)
+            )
+            .padding(all = 10.dp)
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize()
```

Comment:
```suggestion
            modifier = Modifier.fillMaxWidth()
```
Column의 fillMaxSize 같은 경우 부모 size가 정해져있지 않다면 아래와 같이 수직으로 쭉 늘어나기 때문에 보통은 fillMaxWidth만 넣는 경우가 많습니다!
<img width="791" height="1215" alt="image" src="https://github.com/user-attachments/assets/3f90d5bb-904e-46aa-9cf5-b3f13e09bc51" />